README.md

```diff
@@ -1,44 +1,52 @@
-# Office Add-ins Patterns and Practices (PnP)
+# Office Add-ins code samples
```

Comment:
How are these sorted? I'm more curious than anything.

Reply:
Kind of by popularity at the moment. Definitely open to ideas on this. :-)